client/app/scripts/components/node-details/node-details-health-link-item.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import NodeDetailsHealthItem from './node-details-health-item';
+import CloudLink from '../cloud-link';
+import { getMetricColor } from '../../utils/metric-utils';
+import { trackMixpanelEvent } from '../../utils/tracking-utils';
+import { GRAPH_VIEW_MODE } from '../../constants/naming';
+
+export default class NodeDetailsHealthLinkItem extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      hovered: false
+    };
+
+    this.onMouseOver = this.onMouseOver.bind(this);
+    this.onMouseOut = this.onMouseOut.bind(this);
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onMouseOver() {
+    this.setState({hovered: true});
+  }
+
+  onMouseOut() {
+    this.setState({hovered: false});
+  }
+
+  onClick() {
+    trackMixpanelEvent('scope.node.metric.click', {
+      layout: GRAPH_VIEW_MODE,
```

Comment:
The name seems off, maybe `scope.node.details.metric.click`?

Also, the layout could be either table, graph, or resource view, no?